src/main/kotlin/racing/Racing.kt

```diff
@@ -0,0 +1,43 @@
+package racing
+
+import mu.KLogging
+import racing.View.display
+
+class Racing(
+    private val carCnt: CarCnt,
+    private val roundCnt: RoundCnt
+) {
+
+    companion object : KLogging()
+
+    fun start() {
+        game(roundList = roundCnt.toList(), carList = carCnt.toList())
+            .display()
+    }
+
+    fun game(roundList: List<Cnt>, carList: List<Cnt>): List<MutableList<Car>> {
+        val games: MutableList<MutableList<Car>> = mutableListOf()
+
+        roundList.forEachIndexed { index, _ ->
+            val prevGame = kotlin.runCatching { games[index - 1] }.getOrDefault(emptyList())
+
```

Comment:
runCatching 보다는 List의 확장함수를 사용해보는 것은 어떨까요?
```suggestion
        val prevGame = games.getOrElse(index - 1) { emptyList<MutableList<Car>>() }
```

Reply:
@malibinYun 수정했습니다. 피드백 감사합니다.
https://github.com/next-step/kotlin-racingcar/pull/590/commits/68eb5e3b58663bc5d5ab87ccc86e3be338039db2